compose/material/material/src/desktopMain/kotlin/androidx/compose/material/DesktopAlertDialog.desktop.kt

```diff
@@ -182,8 +184,13 @@ object PopupAlertDialogProvider : AlertDialogProvider {
             focusable = true,
             onDismissRequest = onDismissRequest,
         ) {
-            Surface(elevation = 24.dp) {
-                content()
+            Box (
```

Comment:
Better to keep "click outside -> close dialog" feature:
```
            Box (
                modifier = Modifier
                    .fillMaxSize()
                    .pointerInput(onDismissRequest) {
                        detectTapGestures(onPress = { onDismissRequest() })
                    },
                contentAlignment = Alignment.Center
            ) {
                Surface(elevation = 24.dp, onClick = {}, indication = null) {
                    content()
                }
            }
```

Reply:
Done.